qa/workunits/rados/test_rados_tool.sh

```diff
@@ -52,18 +52,30 @@ run() {
 DNAME="`dirname $0`"
 DNAME="`readlink -f $DNAME`"
 RADOS_TOOL="`readlink -f \"$DNAME/../rados\"`"
-if ! test -f $RADOS_TOOL ; then
-    RADOS_TOOL=$(which rados)
+if [ ! -f "$RADOS_TOOL" ] ; then
```

Comment:
i don't think we need to use `detect-build-env-vars.sh` for propagate `$TMPDIR`, and `detect-build-env-vars.sh` is not used to "propagate" env variables, it is used to *detect* the settings not included in the env vars, which are passed down by shell scripts and cmake.